scripts/database_populate_users.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
```

Comment:
database_migration.sh ?

Since it only applies the migration and does not populate the user table